src/main/resources/templates/test.html

```diff
@@ -11,7 +11,7 @@
   <div layout:fragment="content">
     <h1>Test Page</h1>
 
-    <div id="tableContainer"></div>
+    <div id="tableContainer" class="rounded-lg border-grey-200 border-2  overflow-auto"></div>
```

Comment:
* Borders should be applied to the table rather than its container
* `overflow-auto` should go on a div that contains **only** the table, as this container will also contain other elements (paging controls, etc.) that are outside of the table and should not scroll with the table